build.py

```diff
@@ -100,37 +100,49 @@
 #### Kapacitor Functions
 ################
 
+def print_banner():
+    logging.info("""
+                               _             
+  /\\ /\\__ _ _ __   __ _  ___(_) |_ ___  _ __ 
+ / //_/ _` | '_ \\ / _` |/ __| | __/ _ \\| '__|
+/ __ \\ (_| | |_) | (_| | (__| | || (_) | |   
+\\/  \\/\\__,_| .__/ \\__,_|\\___|_|\\__\\___/|_|   
+           |_|                               
+  Build Script
+""")
```

Comment:
Can we make this the same as https://github.com/influxdata/kapacitor/blob/master/cmd/kapacitord/run/command.go#L19